NETWORK/NetworkAddEntityToSynchronisedScene.md

```diff
@@ -5,16 +5,42 @@ ns: NETWORK
 
 ```c
 // 0xF2404D68CBC855FA 0x10DD636C
-void NETWORK_ADD_ENTITY_TO_SYNCHRONISED_SCENE(Entity entity, int netScene, char* animDict, char* animName, float speed, float speedMulitiplier, int flag);
+void NETWORK_ADD_ENTITY_TO_SYNCHRONISED_SCENE(Entity entity, int netScene, char* animDict, char* animName, float blendInSpeed, float blendOutSpeed, float flag);
 ```
 
 
-## Parameters
-* **entity**: 
-* **netScene**: 
-* **animDict**: 
-* **animName**: 
-* **speed**: 
-* **speedMulitiplier**: 
-* **flag**: 
+### Parameters
+* **entity**: The Entity that the animation is gonna play on (Recomended that the entity is serversided for best syncing).
+* **netScene**: scene id.
+* **animDict**: The animDict of the animation that you want to play (remember you have to load with RequestAnimDict() before executing this native)
+* **animName**: The name of the animation that the entity is gonna play.
+* **blendInSpeed**: a float (normal speed is 8.0f).
+* **blendOutSpeed**: a float (normal speed is -8.0f).
+* **flag**: A float 1000.0f (Also doesn't seem to affect anything).
```

Comment:
From my research of this native method, `flag` is still an `int`. Here's what I have:

```c
void __fastcall sub_7FF70329FEAC(signed int entity, unsigned int netsceneId, __int64 animdict, __int64 animname, float blendInSpeed, float blendOutSpeed, int flag)
{
  __int64 _entity; // rax
  int blendInSpeedC; // xmm0_4
  int blendOutSpeedC; // xmm1_4

  _entity = getPhysicalFromScriptHandle(entity);
  if ( _entity && *(_entity + 40) != 4 )
  {
    blendInSpeedC = LODWORD(blendInSpeed);
    if ( blendInSpeed <= 0.0 )
      blendInSpeedC = 0x447A0000;               // ieee-754 (1000.0f)
    blendOutSpeedC = LODWORD(blendOutSpeed);
    if ( blendOutSpeed >= 0.0 )
      blendOutSpeedC = 0xC47A0000;              // ieee-754 (-1000.0f)
    sub_7FF70356F714(_entity, netsceneId, animdict, animname, blendInSpeedC, blendOutSpeedC, flag);
  }
}
```

If `blendInSpeed` is less than `0.0`, it's forced by the game to an [ieee-754](https://babbage.cs.qc.cuny.edu/IEEE-754.old/32bit.html) value of `1000.0` (float).

If `blendOutSpeed` is greater than `0.0`, the game forces the value to `-1000.0` (float).
So judging by this, `blendInSpeed` should be greater than `0.0` and `blendOutSpeed` lower.

You can find more about the ieee-754 specification [here](https://en.wikipedia.org/wiki/IEEE_754), albeit too long.